### DISCUSSION TITLE
Is there a way to query by metadata?

### DISCUSSION THREAD

**Discussion by rstacruz** [zk-org/zk#252] [Help] [answered]
Hey there - `zk` is awesome! Long term Vim user here who dabbled in Obsidian recently, so I'm very stoked to see my note-taking workflow be working in the CLI again.

Is there a way to query by metadata? Eg, if I have notes that look like this:

```markdown
---
index: true
---
...
```

Can I do something like this?

```sh
zk list --match "metadata.index: true"
```

**Comment by mickael-menu** [comment, marked as answer]
There's nothing out of the box yet for metadata filtering, however for this particular example you could use:

```sh
zk list -Mre -m "index: true"
```
